regulations/static/regulations/css/less/module/drawer.less

```diff
@@ -80,37 +82,21 @@ Panel Header
   }
 
   h2 {
-    color: @black;
+    color: @blue;
```

Comment:
I'm pretty sure these shouldn't be blue by default:

Current ATF:
<img width="230" alt="screen shot 2016-06-22 at 7 12 15 pm" src="https://cloud.githubusercontent.com/assets/326918/16286897/5aa5ce9a-38ad-11e6-8921-f7318e80d121.png">

Patch:
<img width="245" alt="screen shot 2016-06-22 at 7 12 02 pm" src="https://cloud.githubusercontent.com/assets/326918/16286907/634ff444-38ad-11e6-9848-c7a600c02f14.png">

Font size has also grown a bit, but I think that's unrelated to this PR